### DISCUSSION TITLE
Strange behaviour at root directory of a drive

### DISCUSSION THREAD

**Discussion by seniordevel** [doublecmd/doublecmd#1912] [General]
In have 5 drives (disks) on Manjaro Linux.
Pressing '..' at a panel, in order to to go one directory up, should guide me to the root directory of a disk step by step.

In short: I expect '..' not to be not shown as I reach the root of a drive!

This is default behaviour with Totalcommander on Windows and on Linux using Dolphin for example (in the latter the 'UP' button  is greyed out and disabled).

With DC I am still able to go up further,
first a list of available drives is shown in my panel,
next I come to the root directory of my system drive.
Quite irritating for me.

My question:
Is this default behaviour of DC -or- did I mess up something in DC's settings ?

Thanks for your attention.

**Comment by alexx2000** [comment]
Under Linux `/` is a global root. And disks are mounted in the regular directories. So drive root always has an `..` item (except root drive).

**Comment by seniordevel** [comment]
@alexx2000 
Correct, but even most Linux file managers do not show this strange behaviour.
Same with managers on Android, also based on Linux, those stop at the root of any device.

Thus I intend to create an issue, at least as feature request, to add an option for the mentioned behaviour.

Thanks for your opinion.

**Comment by alexx2000** [comment]
DC is a more low-level file manager than others (it shows a file system as is). I don't plan to change this behavior.

**Comment by seniordevel** [comment]
@alexx2000 
I do not ask to change the behaviour, I kindly asked for an option, a setting.
I even do not need a GUI setting, just a setting in the config file.

But nevertheless thanks for your attention.
